compose.yaml

```diff
@@ -8,6 +8,8 @@ services:
     depends_on:
       redis:
         condition: service_started
+      fcp-defra-id-stub:
+        condition: service_healthy
```

Comment:
Wait for server running stub to report healthy before starting frontend.  

This avoids a race condition where frontend request to well known endpoint fails as stub server not yet ready.